test/e2e/infra/vagrant/playbook/roles/common/tasks/busybox.yml

```diff
@@ -1,11 +1,2 @@
 - name: Pull busybox:latest docker image (Ansible < '2.8.0')
-  docker_image:
-    name: busybox:latest
-    pull: yes
-  when: ansible_version.full is version_compare('2.8.0', '<')
-
-- name: Pull busybox:latest docker image (Ansible >= '2.8.0')
-  docker_image:
-    name: busybox:latest
-    source: pull
-  when: ansible_version.full is version_compare('2.8.0', '>=')
+  ansible.builtin.command: ctr image pull docker.io/library/busybox:latest
```

Comment:
could you check if we need `-n=k8s.io` if we want the image to be available in K8s? I suggest you create a simple Pod with `image: docker.io/library/busybox:latest` and `imagePullPolicy: IfNotPresent`, you can then check the Pod with `kubectl describe` to see if the image was pulled or not.